### PR TITLE
flip extended help interface

### PR DESCRIPTION
**Pull request by mmaxim**
Add a help interface for the flip command, but that could be used by any command.

### CODE REVIEW

File: go/chat/types/types.go [hunk 1]
```diff
@@ -402,3 +402,22 @@ func (d DummyStellarSender) DecorateWithPayments(ctx context.Context, body strin
 	payments []chat1.TextPayment) string {
 	return body
 }
+
+type DummyCoinFlipManager struct{}
+
```
Comment (joshblum) [inline go/chat/types/types.go]: ` var _ CoinFlipmanager = (*DummyCoinFlipManager)(nil)`

File: shared/reducers/chat2.js [hunk 2]
```diff
@@ -445,6 +445,10 @@ const rootReducer = (
       })
       return state.set('flipStatusMap', fm)
     }
+    case Chat2Gen.setCommandMarkdown: {
+      const {conversationIDKey, text} = action.payload
+      return state.setIn(['commandMarkdownMap', conversationIDKey], text)
```
Comment (buoyad) [inline shared/reducers/chat2.js]: Can we clear this on `messageSend`?

File: shared/chat/conversation/command-markdown/index.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      backgroundColor: Styles.globalColors.lightGrey,
+      borderRadius: Styles.borderRadius,
+      marginLeft: 15,
+      marginRight: 15,
+      maxHeight: 300,
+    },
+    isMobile: {
+      borderColor: Styles.globalColors.black_10,
+      borderStyle: 'solid',
+      borderWidth: 1,
```
Comment (buoyad) [inline shared/chat/conversation/command-markdown/index.js]: :sheep: `borderTopWidth: 3` will match the suggestion overlay.

File: shared/chat/conversation/command-markdown/container.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+type OwnProps = {|conversationIDKey: Types.ConversationIDKey|}
+
+const mapStateToProps = (state, ownProps: OwnProps) => ({
+  text: state.chat2.commandMarkdownMap.get(ownProps.conversationIDKey, ''),
+})
+
+export default namedConnect<OwnProps, _, _, _, _>(
+  mapStateToProps,
+  (d, o) => ({}),
+  (s, d, o) => ({...s}),
```
Comment (buoyad) [inline shared/chat/conversation/command-markdown/container.js]: `(s, d, o) => s,`

File: shared/reducers/chat2.js (hunk 2 above)
Comment (mmaxim) [reply shared/reducers/chat2.js]: Sure

**Comment by mmaxim** [comment]
Thanks @buoyad, PTAL!

**Pull request merged**